hist/histv7/test/biniter.cxx

```diff
@@ -45,7 +45,7 @@ TEST(BinIterNBins, BinRef) {
   double founduncert = -1.;
   RH2F::CoordArray_t foundcoord{};
 
-  int nBins = 0;
+  int nBins = 1;
```

Comment:
Thanks, Matthias! Would it make sense to fill the histogram say at bin 5, and check that the bin iter `+5` gives access to the correct bin content?

Reply:
That is what the line
EXPECT_EQ(expectedLinBin, foundbin);
does.
Before the fixes the test started at zero instead of one so it was off by one. But the iterator was also starting at zero instead of one. So they where both wrong the same way, so the check passed despite the bugs. :(